executor/executor_test.go

```diff
@@ -236,6 +240,13 @@ func (s *testSuite) TestAdmin(c *C) {
 	r, err = tk.Exec("admin check table admin_test")
 	c.Assert(err, NotNil)
 
+	os.Setenv("TIDB_CHECK_BEFORE_DROP", "1")
+	r, err = tk.Exec("drop table admin_test")
+	c.Assert(err, NotNil)
```

Comment:
Please check the content of the error to make sure it is thrown by the consistent checking.